src/bootstrap.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { createRoot } from 'react-dom/client';
 import AppEntry from './AppEntry';
 
-const root = document.getElementById('root');
+const rootElm = document.getElementById('root');
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+const root = createRoot(rootElm!);
```

Comment:
wasn't sure what a good way to go about this was given the non-null operator was the suggested approach by react 18 (https://react.dev/blog/2022/03/08/react-18-upgrade-guide#updates-to-client-rendering-apis)